client/web/compose/src/views/Admin/Pages/Edit.vue

```diff
@@ -244,6 +249,22 @@ export default {
     showDeleteDropdown () {
       return this.hasChildren && this.page.canDeletePage && !this.page.deletedAt
     },
+
+  },
+
+  watch: {
+    pageID: {
+      immediate: true,
+      handler (pageID) {
+        if (pageID) {
```

Comment:
Make sure you do it how it was done in created, since currently you're double fetching it. 
And the catch is missing, so yes. Look at created